tests/test_basis_utils.py

```diff
@@ -9,10 +10,56 @@
     norm_assoc_legendre,
     real_sph_harmonic,
     sph_bessel,
+    sph_harm,
     unique_coords_nd,
 )
 
 
+def test_sph_harm_low_order():
+    """
+    Test the `sph_harm` implementation matches `scipy` at lower orders.
+    """
+    m = 3
+    j = 5
```

Comment:
Should we also test the branch where `m < 0`?

Reply:
We don't use, but yes, I had a typo there... fixed.